OpenMEEG/libs/OpenMEEG/src/mesh.cpp

```diff
@@ -975,7 +977,7 @@ namespace OpenMEEG {
         os.write((char*)normals_raw, sizeof(float)*nb_vertices()*3);       // normals
         unsigned char zero[1] = {0};
         os.write((char*)zero, sizeof(unsigned int));                       // arg size : 0
-        unsigned int faces_number[1] = {nb_triangles()};
+        unsigned int faces_number[1] = {(unsigned)nb_triangles()};
```

Comment:
yes ... first I used the same function sizet_to<unsigned> for that, but then on Windows, it created problems with __declspec(dllexport) ...  so I'll use only a static cast here (without a check). is it okay for you ?